agenda-services/src/main/java/org/exoplatform/agenda/util/NotificationUtils.java

```diff
@@ -682,9 +682,8 @@ public static String getRecurrenceDetails(Event event) {
           }
         case "WEEKLY":
           if (eventRecurrence.getInterval() == 1 && event.getRecurrence().getByDay().size() == 1) {
-            String dayNumber = eventRecurrence.getByDay().get(0);
-            DayOfWeek dayName = DayOfWeek.of(Integer.parseInt(dayNumber));
-            return "Weekly on " + StringUtils.lowerCase(String.valueOf(dayName)) + "";
```

Comment:
you should put this same code without event.getRecurrence().getByDay().size() conditions. Same for the next eventRecurrence.getInterval() > 1 condition